migrations/migrations/20190508143516-create-user.js

```diff
@@ -7,30 +7,31 @@ module.exports = {
         allowNull: false,
         primaryKey: true
       },
-      first_name: {
+      name: {
         type: Sequelize.STRING,
         allowNull: false
       },
       last_name: {
         type: Sequelize.STRING,
         allowNull: false
       },
-      username: {
-        type: Sequelize.STRING,
-        allowNull: false,
-        unique: true
-      },
       email: {
         type: Sequelize.STRING,
-        allowNull: false,
-        unique: true
+        unique: true,
```

Comment:
create a new migration please

Reply:
Do I delete the old one then?

Reply:
I deleted the old one and created a new one, now the date in the name is correct. I chose to do that because I thought that creating a new migration to modify the table was not necessary because the old one came with the bootstrap, and didn't have any use up until this point in the project. What do you think? do you want me to do one specifically to change the table or do you think what I did makes sense?

Reply:
Done, sorry for the delay, I was a bit conflicted